docs/source/contributing/ci.adoc

```diff
@@ -39,6 +39,8 @@ Thus it is used to run integration tests in addition to unit tests.
 The artifacts of a successful master build can be found at link:http://artifacts.ci.centos.org/minishift/minishift/master/[artifacts.ci.centos.org/minishift/minishift/master/<BUILD_ID>].
 The artifacts of a successful pull request build can be found at link:http://artifacts.ci.centos.org/minishift/minishift/pr/[artifacts.ci.centos.org/minishift/minishift/pr/<PR_ID>].
 
+As part of master build, a {project} documentation integrated with OpenShift documentation is hosted at link:http://artifacts.ci.centos.org/minishift/minishift/docs/master/openshift-origin/latest/minishift/index.html[artifacts.ci.centos.org/minishift/minishift/docs/master/openshift-origin/latest/minishift/] which reflects the documentation from the _master_ branch.
```

Comment:
"As part of a master build, {project} documentation integrated..."

I'm also not particularly fond of having "which reflects documentation from the _master_ branch" at the end of the sentence here. It's ambiguous -- are you talking about Minishift documentation from our master branch, OpenShift documentation from their master branch, or both?

We can slightly rewrite this if I know which of those is the most technically accurate.

Reply:
@robin-owen Slightly changed the documentation. See below

![screenshot from 2017-12-04 11-59-11](https://user-images.githubusercontent.com/1132451/33540351-bdc0b460-d8f0-11e7-913e-b8efda536695.png)